sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -144,12 +144,6 @@ export class MessageReceiver extends LinkEntity {
    * Default: `300 * 1000` (5 minutes);
    */
   maxAutoRenewDurationInMs: number;
-  /**
```

Comment:
I do love changes that are all deletions!

Reply:
Yeah, there was some shared code that never actually got used. I was happy to see it go as well :)